prow/repoowners/endpoint_test.go

```diff
@@ -0,0 +1,154 @@
+package repoowners
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func generateQueryBody(body *ownersRequest) []byte {
+	rawBody, _ := json.Marshal(body)
+	return rawBody
+}
+
+func TestEndpoint(t *testing.T) {
+	tests := []struct {
+		name         string
+		body         []byte
+		contentType  string
+		expectedBody string
+		expectedCode int
+	}{
+		{
+			name: "general use case",
+			body: generateQueryBody(&ownersRequest{
+				Org:        "org",
+				Repo:       "repo",
+				BaseCommit: "Master",
```

Comment:
Nit: maybe just use `master` (same point for the other test-cases below).

Reply:
This is coming from the repoowners test